Tests/Functional/Performance/Misuse/main.swift

```diff
@@ -0,0 +1,135 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/PerformanceMisuse
+// RUN: %{built_tests_dir}/PerformanceMisuse > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(Linux) || os(FreeBSD)
+    import XCTest
+#else
+    import SwiftXCTest
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'PerformanceMisuseTestCase' started at \d+:\d+:\d+\.\d+
+class PerformanceMisuseTestCase: XCTestCase {
+
+    // CHECK: Test Case 'PerformanceMisuseTestCase.test_whenMeasuringMultipleInOneTest_fails' started at \d+:\d+:\d+\.\d+
+    func test_whenMeasuringMultipleInOneTest_fails() {
+        // CHECK: .*/Misuse/main.swift:20: Test Case 'PerformanceMisuseTestCase.test_whenMeasuringMultipleInOneTest_fails' measured.*
```

Comment:
Including the line numbers already caught regressions on my part a couple of times while I was working on this 👍